### PR TITLE
fix: update missing or incorrect enum values

### PR DESCRIPTION
**Pull request by jessicamcinchak**
- Adds `other` option to `data.applicant.interest`
- Adds `proposal.photograph.evidence` to `FileType`
- Updates `property.site.locationPlan` &rarr; `property.drawing.locationPlan` in `FileType`
- Updates descriptions only for elevations & sections in `FileType`

Fixes validation errors thrown for a number of past applications checked here: https://github.com/theopensystemslab/planx-core/pull/166

### CODE REVIEW

File: types/schema/data/Property.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export interface UKProperty {
   localAuthorityDistrict: string[];
   type: PropertyType;
   boundary?: {
-    site: GeoJSON;
-    area: Area;
+    site?: GeoJSON;
+    area?: Area;
```
Comment (DafyddLlyr) [inline types/schema/data/Property.ts:22]: This is the only change I don't quite understand sorry.

Is it not the case that a boundary is either `undefined` if they upload a location plan, or all fields are required if they draw a boundary, i.e. `{ site: GeoJSON, area: Area }`

**Review by DafyddLlyr — APPROVED** [review]
One small question - I might just be missing something here?

File: types/schema/data/Property.ts (hunk 1 above)
Comment (jessicamcinchak) [reply types/schema/data/Property.ts:22]: That was also my impression of how it should be working, but empty object fields still seem to be coming through - so rather than just omitting the `boundary` key altogether it's coming through as `boundary: {}`, which was then incorrectly expecting nested keys to exist. 

Need to properly test / fix these cases (it's more than just boundary) in the passport mapping in planx-core first then can come back to tighten these types again.

Comment (DafyddLlyr) [reply types/schema/data/Property.ts:22]: Yep - that's totally fair and makes sense if the plan is to come back and tighten up 👍 Maybe a `todo` here is a good shout?

Comment (jessicamcinchak) [reply types/schema/data/Property.ts]: Decided to revert this before merging, think I'll have it worked out in the next planx-core change :+1:
